demo-server.html

```diff
@@ -149,9 +147,9 @@
 <div class="warning">
 <p class="warning-title">Warning!</p>
 
-<p class="warning-text">Due to the steady increase of issues not under OMERO's control, continued support of Java Web Start for distribution of OMERO.insight is not sustainable and is likely to become impossible in the near future.</p>
+<p class="warning-text">Due to the steady increase of issues not under OMERO's control, continued support of Java Web Start for distribution of OMERO.insight has become impossible.</p>
```

Comment:
I think I'd be inclined to just remove this mention of Web Start altogether - if someone is just signing up for the demo now, they won't know anything about it anyway and pointing out that it doesn't exist any more seems rather odd.